extension-client/src/command/shell/tfcost/index.ts

```diff
@@ -0,0 +1,22 @@
+import * as shell from '..';
+import * as util from '../../../util';
+var os = require('os');
+
+const TFCOST_COMMAND = 'tfcost';
+
+export function calculateTFCost(key: string): Promise<string | Error> {
```

Comment:
Similarly we could rename `calculateTFCost` to just `calculateCost` since we know it's under tfcost folder